app/src/main/java/org/apache/commons/text/similarity/FuzzyScore.java

```diff
@@ -84,7 +84,7 @@ public FuzzyScore(final Locale locale) {
      * @return result score
      * @throws IllegalArgumentException if the term or query is {@code null}
      */
-    public Integer fuzzyScore(final CharSequence term, final CharSequence query) {
+    public int fuzzyScore(final CharSequence term, final CharSequence query) {
```

Comment:
I think this was taken from [here](https://commons.apache.org/sandbox/commons-text/jacoco/org.apache.commons.text.similarity/FuzzyScore.java.html). Does [the apache license](http://www.apache.org/licenses/LICENSE-2.0) allow modifying code? If so, does it require changes to be documented?

Reply:
I undid the change.